config/notifiers.go

```diff
@@ -742,6 +742,7 @@ type PushoverConfig struct {
 	Expire      duration `yaml:"expire,omitempty" json:"expire,omitempty"`
 	TTL         duration `yaml:"ttl,omitempty" json:"ttl,omitempty"`
 	HTML        bool     `yaml:"html" json:"html,omitempty"`
+	Monospace   bool     `yaml:"monospace" json:"monospace,omitempty"`
```

Comment:
It looks like we are missing `omitempty` on the `yaml` tag.

Reply:
Should the HTML yaml tag have an `omitempty` as well? It currently does not have it, so I kept this in line with the other bool.